core/services/chainlink/application.go

```diff
@@ -936,7 +937,7 @@ func (app *ChainlinkApplication) stop() (err error) {
 			err = multierr.Append(err, app.profiler.Stop())
 		}
 
-		app.logger.Info("Exited all services")
+		app.logger.Debugf("Closed application in %v", time.Since(shutdownStart))
```

Comment:
Is this measuring something different than this line? https://github.com/smartcontractkit/chainlink/blob/2dadb614a3fbeecf32e17ca6c374c64af3a69c9f/core/cmd/shell_local.go#L345

Reply:
Didn't see that one. Not really, measures something very similar just without the DB close and it will appear in structured logging